ui/public/i18n/en/translation.json

```
@@ -24,7 +24,36 @@
     "title": "Settings",
     "configure_instance": "Configure {instance}",
     "save": "Save",
-    "test_field": "Test field"
+    "receiver_emails": "Email notifications",
+    "receiver_emails_list": "List of email recipients for notifications",
+    "whitelists": "Allow list of CIDR, IP and fully qualified domain name",
+    "receiver_emails_placeholder": "One email address per line",
+    "whitelist_placeholder": "One entry per line",
+    "advanced": "Advanced",
+    "ban_local_network": "Allow bans on the local network",
+    "ban_local_network_tips": "By default no ban occurs on private IP class",
+    "enabled": "Enabled",
+    "disabled": "Disabled",
+    "dyn_bantime": "Dynamic ban time",
+    "dyn_bantime_tips": "Ban time is calculated by ((number of attempts + 1) * 4) minutes",
+    "bantime": "Bantime",
+    "minutes": "Minutes",
+    "disable_online_api": "Disable central API",
+    "disable_online_api_tips": "The central API is used to retrieve blacklists and to have a glance on metrics at app.crowdsec.net",
+    "open_app_crowdsec": "App crowdsec website",
+    "enroll_instance": "Enroll the crowdsec instance",
+    "enroll_instance_must_be_real_token": "The token must be retrieved on the website app.crowdsec.net",
+    "enroll_instance_placeholder": "xxxxxxxxxxxxxxxxxxx",
+    "helo_host": "Helo_host",
+    "helo_host_must_be_relevant_for_smtp": "You can set a specific Helo FQDN if your antispam adds score due to default localhost Helo name",
+    "helo_host_placeholder": "sub.domain.com",
+    "enable_smarthosts_for_notifications": "Smarthost settings",
+    "bad_email_address": "Bad email address",
+    "bad_IP_or_hostname": "Bad CIDR, IP or FQDN",
+    "processing": "Processing...",
+    "required": "Required",
+    "token_not_valid":"The attachment key provided is not valid",
```

Let's always call it "token"
```suggestion
    "token_not_valid":"The token provided is not valid",
```